enarx-keep/src/backend/kvm.rs

```diff
@@ -42,10 +47,33 @@ impl backend::Backend for Backend {
     }
 
     fn shim(&self) -> Result<PathBuf> {
-        unimplemented!()
+        #[cfg(debug_assertions)]
+        const PROFILE: &str = "debug";
+
+        #[cfg(not(debug_assertions))]
+        const PROFILE: &str = "release";
+
+        let path = PathBuf::from(env!("CARGO_MANIFEST_DIR"))
```

Comment:
I would expect some debt issue to make this all configurable, instead of hardcoding the paths from the compilation layout.